transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -530,9 +557,11 @@ private void fireContentReceived(DefaultHttpSession session, HttpContentMessage
 
     @Override
     public void addBridgeFilters(IoFilterChain chain) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "addBridgeFilters" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)